README.md

```diff
@@ -53,6 +53,123 @@ See [RADIUS.md](RADIUS.md) for a list of supported Attrbiutes.
 
 Many thanks to [On Waves](https://www.on-waves.com/) for sponsoring the RADIUS Authentication implementation.
 
+Example of configuration **RADIUS**:
+```erlang
+%% ...
+{ergw_aaa, [
+    {handlers, [
+        {ergw_aaa_static, [
+            {'Node-Id',        <<"CHANGE-ME">>},            %% <- CHANGE
+            {'NAS-Identifier', <<"CHANGE-ME">>},            %% <- CHANGE
+            {'NAS-IP-Address', {127,0,0,3}},                %% <- CHANGE
+            {'Acct-Interim-Interval',   1800},              %% <- CHANGE
+            {'Framed-Protocol',         'PPP'},
+            {'Service-Type',            'Framed-User'}
+        ]},
+        {ergw_aaa_radius, [
+            {server,
+                {{127,0,0,4}, 1813, <<"CHANGE-ME-SECRET">>} %% <- CHANGE IP and SECRET
+            }
+        ]}
+    ]},
+    {services, [
+        {'Default', [
+            {handler, 'ergw_aaa_static'}
+        ]},
+        {'RADIUS-Acct', [
+            {handler, 'ergw_aaa_radius'}
+        ]}
+    ]},
+    {apps, [
+        {default, [
+            {session, ['Default']},
+            {procedures, [
+                {authenticate, []},
+                {authorize, []},
+                {start, ['RADIUS-Acct']},
+                {interim, ['RADIUS-Acct']},
+                {stop, ['RADIUS-Acct']},
+                {{gx, 'CCR-Initial'},   []},
+                {{gx, 'CCR-Update'},    []},
+                {{gx, 'CCR-Terminate'}, []}
```

Comment:
It might be worth noting that this does not result in a working setup. We use the Gx API to configure the policies. Either a real Gx server or a fake answer is needed to get working policies.